src/_designs/components/label/index.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <span
+    class="label-wrapper"
+    :class="{ [fill]: Boolean(fill)
```

Comment:
either keep this inline or collapse it like:

```
:class="{ 
   [fill]: Boolean(fill)
}
```

Reply:
updated